lib/github-pages/configuration.rb

```diff
@@ -19,6 +19,7 @@ class Configuration
       "kramdown" => {
         "input"     => "GFM",
         "hard_wrap" => false,
+        "gfm_quirks" => "paragraph_end",
```

Comment:
Should this simply be an enforced override? Under what circumstances would a user want to turn this feature off?